tahoe_sites/tests/test_api.py

```diff
@@ -520,3 +521,70 @@ def test_get_organization_by_course_only_one_active(self):
         self.add_organization_course(organization=self.default_org)
 
         assert api.get_organization_by_course(course_id='dummy_key') == self.default_org
+
+    @ddt.data(True, False)
+    def test_get_organization_user_by_email(self, user_is_active):
+        """
+        Verify that get_organization_user_by_email returns the correct user
+        """
+        self._prepare_mapping_data()
+        self.default_user.is_active = user_is_active
+        self.default_user.save()
+        email = self.default_user.email
+
+        # The email is related to default_org
+        assert api.get_organization_user_by_email(email=email, organization=self.default_org) == self.default_user
+        with self.assertRaises(get_user_model().DoesNotExist):
```

Comment:
`pytest` usually gives a much better debugging messages

```suggestion
        with pytest.raises(get_user_model().DoesNotExist):
```

Reply:
done